packages/aws-cdk-lib/aws-stepfunctions/lib/states/distributed-map/item-batcher.ts

```diff
@@ -71,7 +73,7 @@ export class ItemBatcher {
       ...(this.props.maxItemsPerBatchPath && { MaxItemsPerBatchPath: this.props.maxItemsPerBatchPath }),
       ...(this.props.maxInputBytesPerBatch && { MaxInputBytesPerBatch: this.props.maxInputBytesPerBatch }),
       ...(this.props.maxInputBytesPerBatchPath && { MaxInputBytesPerBatchPath: this.props.maxInputBytesPerBatchPath }),
-      ...(this.props.batchInput && { BatchInput: this.props.batchInput }),
+      ...(this.props.batchInput && { BatchInput: FieldUtils.renderObject(this.props.batchInput) }),
```

Comment:
I would like to understand more on why do we need to render only this specifc `batchInput` property since the existing behavious itself will be able to pass Json payload to the `batchInput` property?

Reply:
+1 to the question above and also to provide some more details in the description.